Templating/Helper/DateTimeHelper.php

```diff
@@ -46,10 +46,10 @@ public function __construct(TimezoneDetectorInterface $timezoneDetector, $charse
     }
 
     /**
-     * @param \Datetime|string|int $date
-     * @param null|string          $locale
-     * @param null|string          timezone
-     * @param null|int             dateType See \IntlDateFormatter::getDateType
+     * @param \DateTimeInterface|string|int $date
```

Comment:
I think it should be `\DateTime|\DateTimeInterface` with a `NEXT_MAJOR` task.

http://php.net/manual/fr/class.datetimeinterface.php -> Only since PHP 5.5